tests/baselines/reference/objectLiteralExcessProperties.errors.txt

```diff
@@ -40,7 +50,7 @@ tests/cases/compiler/objectLiteralExcessProperties.ts(33,27): error TS2322: Type
     var b2: Book | string = { foreward: "nope" };
                               ~~~~~~~~~~~~~~~~
 !!! error TS2322: Type '{ foreward: string; }' is not assignable to type 'string | Book'.
-!!! error TS2322:   Object literal may only specify known properties, and 'foreward' does not exist in type 'string | Book'.

```

Comment:
Side note: in this or another PR, we should make these both `but`

Reply:
It's probably most correct to have two separate sentences, as the subjects of the clauses are not well-connected. `Object literal may only specify known properties. 'foreward' does not exist in type 'string | Book'.` Two separate statements of fact - there is no _real_ relationship between them, so a conjunction feels a bit extraneous to me. 🚲 🏚 

I call in @sandersn as our resident linguist to weigh in.